packages/flutter/lib/src/material/input.dart

```diff
@@ -166,6 +155,8 @@ class _InputState extends State<Input> {
 
     _attachOrDetachKeyboard(focused);
 
+    _editableString.setText(config.value.text);
```

Comment:
We shouldn't need to do this every time we build.  We should be able to do this in `didUpdateConfig`.